theano/configdefaults.py

```diff
@@ -1259,21 +1262,23 @@ def default_blas_ldflags():
             else:
                 # This branch is executed if no exception was raised
                 lib_path = os.path.join(sys.prefix, 'DLLs')
-                flags = ['-L%s' % lib_path]
+                flags = ['-L"%s"' % lib_path]
                 flags += ['-l%s' % l for l in ["mkl_core",
                                                "mkl_intel_thread",
                                                "mkl_rt"]]
                 res = try_blas_flag(flags)
                 if res:
                     return res
 
+        # to support path that includes spaces, we need to wrap it with double quotes on Windows
+        path_wrapper = "\"" if os.name == 'nt' else ""
```

Comment:
Honestly, I am not a huge specialist in platform related issues. The reasons I choose to use os.name == 'nt'condition were:
1. This is the condition used in file **theano/misc/windows.py** (line 13)
2. This is the fist solutions that pops-up with google search "python check if windows" -> 
go to http://stackoverflow.com/questions/1325581/how-do-i-check-if-im-running-on-windows-in-python (first link in search) and then the most voted solution (105 votes) suggests using os.name, and in the next solution (with sys.platform) you can find _'But os.name is probably the way to go, as others have mentioned.'_
I googled the difference between both, and as far as I understand the os.name is more general than the sys.platform. So, os.name could be used to check if I run on **any** Windows OS, and sys.platform could be used to check if I run on **some specific** Windows platform. But, once again, I am not a specialist in OS. 
I believe the paths should be treated the same across all Windows platforms (alike subprocess is treated the same in windows.py), so os.name is more appropriate here. sys.platform, may be more appropriate for compilation related checks, though.